cmd/ebrelayer/txs/relayToCosmos.go

```diff
@@ -23,8 +20,8 @@ var (
 
 // RelayToCosmos applies validator's signature to an EthBridgeClaim message containing
 // information about an event on the Ethereum blockchain before relaying to the Bridge
-func RelayToCosmos(cdc *codec.Codec, moniker, password string, claims []types.EthBridgeClaim, cliCtx context.CLIContext,
-	txBldr authtypes.TxBuilder, sugaredLogger *zap.SugaredLogger) error {
+func RelayToCosmos(moniker, password string, claims []*types.EthBridgeClaim, cliCtx client.Context,
```

Comment:
Same question here: Why do we need to make it an array of pointers to EthBridgeClaims instead of just an array of EthBridgeClaims?